### PR TITLE
Issue#11 - Add "--turns" option to allow setting the number of turns for test battles

### PR DESCRIPTION
**Pull request by tenshi13**
2nd attempt at https://github.com/JSJitsu/hero-starter/issues/11
- added commander.js library to parse cli parameters
  - https://github.com/tj/commander.js
- adjusted logic for "turns" input

### CODE REVIEW

File: test-battle.js [hunk 1]
```diff
@@ -18,18 +18,28 @@ To run:
...
+
+// accepting cli parameters
+var args = require('commander');
+args.description('CLI to test the hero.js code locally')
+    .option('-w, --wait', 'Turn by turn step through of the battle')
+    .option('-t, --turns [int]', 'Specifies how many turns to run', parseInt)
+    .parse(process.argv);
+
+// args validation
+if ( isNaN(args.turns) ){
```
Comment (Asuza) [inline test-battle.js]: I just realized while testing these changes that this condition prevents you from running the battle with default settings. I think this needs to be updated to:

``` javascript
if (args.turns && isNaN(args.turn)) {
```

**Comment by Asuza** [comment]
It's looking good, @tenshi13. The CLI options are nicely organized. Good choice on the package!

**Comment by tenshi13** [comment]
@Asuza hey man, sorry for that, thanks for the review, i've did an update, can you please review it again?

tq

**Comment by Asuza** [comment]
@tenshi13, I apologize for the delay. I knew I was forgetting a PR somewhere.

**Comment by tenshi13** [comment]
@Asuza  no worries man, glad its been accepted ; D

**Pull request merged**